### PR TITLE
Whitelist dbacademy

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
Whitelist dbacademy

### Linked issues
Progresses #1901 

### Functionality
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -1690,6 +1690,276 @@
...
+      {
+        "code": "dbfs-usage",
+        "message": "Deprecated file system path: dbfs:/FileStore/tmp/"
+      },
+      {
+        "code": "dbfs-usage",
+        "message": "Deprecated file system path: dbfs:/mnt/resources.training.databricks.com/distributions/"
+      }
+    ],
+    "dbacademy.dbbuild.test": [],
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:1866]: Remove tests

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -1690,6 +1690,276 @@
     "datasetsforecast.m5": [],
     "datasetsforecast.utils": []
   },
+  "dbacademy": {
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:1693]: I can not find this package on Pypi" https://pypi.org/search/?q=dbacademy. Is it a relative package?

**Review by JCZuurmond — CHANGES_REQUESTED** [review]
see comments

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:1693]: no you install it using a git url

Comment (JCZuurmond) [reply src/databricks/labs/ucx/source_code/known.json:1693]: Could you add the link? As a comment in the know.json?

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:1866]: done

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:1693]: json does not support comments, the link is here https://github.com/databricks-academy/dbacademy/blob/main/README.md

**Pull request merged**